src/components/PsmPanel.tsx

```diff
@@ -45,12 +43,15 @@ const ProposalTypes = {
 export default function PsmPanel() {
   const [anchorName, setAnchorName] = useState(anchors[0]);
   const [proposalType, setProposalType] = useState(ProposalTypes.paramChange);
-  const walletUtils = useContext(WalletContext);
-  const { data: walletCurrent } = usePublishedDatum(
-    `wallet.${walletUtils.getWalletAddress()}.current`,
+  const walletUtils = useAtomValue(walletUtilsAtom);
+  const { data: walletCurrent, status } = usePublishedDatum(
+    walletUtils
+      ? `wallet.${walletUtils.getWalletAddress()}.current`
+      : undefined,
   );
```

Comment:
ditto for the other usePublishedDatum with a ternary

Reply:
`usePublishedDatum` contains a `useEffect` and React requires `useEffect` hooks to be called consistently on every render. If we call it conditionally on different renders it will error: https://react.dev/reference/rules/rules-of-hooks#only-call-hooks-at-the-top-level

Allowing an `undefined` path allows the hook to be called every time, even if the component doesn't have a path to watch yet, and the undefined path is handled within `useEffect`. This mechanism was added in https://github.com/Agoric/dapp-econ-gov/pull/126